source/administration/command-line-tools.rst

```diff
@@ -680,6 +741,170 @@ mattermost export schedule
           --exportFrom string     Unix timestamp (seconds since epoch, UTC) to export data from.
           --timeoutSeconds string Set how long the export should run for before timing out.
 
+mattermost group  
+------------------------
+
+  Description
+    Commands for managing Mattermost groups.  For more information on Mattermost groups please see `this documentation. <https://docs.mattermost.com/deployment/ldap-group-sync.html>`_
+
+  Child Commands
+    -  `mattermost group channel`_ - Management of Mattermost groups linked to channels
+    -  `mattermost group team`_ - Management of Mattermost groups linked to teams
+    
+mattermost group channel 
+------------------------
+
+  Description
+    Commands for managing Mattermost groups linked to a channel.
+
+  Child Commands
+    -  `mattermost group channel enable`_ - Enables group constraint on the specified channel
+    -  `mattermost group channel disable`_ - Disables group constraint on the specified channel
+    -  `mattermost group channel list`_ - Lists the groups associated with a channel
+    -  `mattermost group channel status`_ - Shows the group constraint status of the specified channel
+
+mattermost group channel enable 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  Description
+    Enables group constraint on the specified channel.  When a channel is group constrained, channel membership is managed by linked groups instead of managed by manually adding and removing users.
```

Comment:
Should we mention here that a channel needs to have at least one group associated for this command to work?

Reply:
Also nit, double space:

```suggestion
    Enables group constraint on the specified channel. When a channel is group constrained, channel membership is managed by linked groups instead of managed by manually adding and removing users.
```